sign/dilithium/dilithium.go

```diff
@@ -6,13 +6,23 @@
 // https://pq-crystals.org/dilithium/data/dilithium-specification-round2.pdf
 //
 // Each of the eight different modes of Dilithium is implemented by a
-// subpackge.  For instance, Dilithium III can be found in
+// subpackage.  For instance, Dilithium III can be found in
 //
 //  github.com/cloudflare/circl/sign/dilithium/mode3
 //
 // If your choice for mode is fixed compile-time, use the subpackages.
 // This package provides a convenient wrapper around all of the subpackages
 // so one can be chosen at runtime.
+//
+// As we don't understand the security of Dilithium well yet, it is advisable
```

Comment:
From https://pq-crystals.org/dilithium/
```
For users who are interested in using Dilithium, we recommend the following:

Use Dilithium in a so-called hybrid mode in combination with an established "pre-quantum" signature scheme.
We recommend using the Dilithium-1280x1024 parameter set, which—according to a very conservative analysis—achieves around 128 bits of security against all known classical and quantum attacks.
```

Reply:
What about the current version?

Reply:
It's ok. Do the rebase with master and proceed to merge.